parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -155,6 +163,9 @@ pub mod pallet {
 		InvalidExecutionHeaderUpdate,
 		FinalizedBeaconHeaderSlotsExceeded,
 		ExecutionHeaderMappingFailed,
+		FinalityHeaderBoundsExceed,
+		ExecutionHeaderBoundsExceed,
+		SyncCommitteeBoundsExceed,
```

Comment:
These need an 'ed' at the end.
```suggestion
		FinalityHeaderBoundsExceeded,
		ExecutionHeaderBoundsExceeded,
		SyncCommitteeBoundsExceeded,
```